README.md

```diff
@@ -109,7 +109,7 @@ Below are links for post-installation tweaks for Windows 10+.
 
 We do encourage you to download and set your background to the FLARE VM logo!
 <p align="center">
-  <img width="300" height="300" src="flarevm.png?raw=true" alt="FLARE VM"/>
+  <img width="300" height="300" src="flarevm-logo.png?raw=true" alt="FLARE VM"/>
```

Comment:
This change does not fix the broken image in the README. Note that the FLARE-VM background image is stored in `Images/flarevm-background.png` and the FLARE-VM logo in `Images/flarevm-logo.png`. The currently installer does set the background image and it seems to work fine in Windows 10. So I am not sure if we need this information. I would like to set the lock screen as well.